app/views/survey/demographic_questions/_wwf_programme.html.erb

```diff
@@ -1,28 +0,0 @@
-<p class="card-text">
```

Comment:
I would keep this file in actually, if we're taking the policy of just unpublishing the question, I think it's a good reference to have. Plus it gives us the option to reenable it should we want to

Reply:
Great idea I felt like deleting it was a bad idea. So I will restore it.